javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,90 @@
+const inputEl = document.getElementById('inputEl'); 
+const convertFrom = document.getElementById('convertFrom'); 
+const convertTo = document.getElementById('convertTo'); 
+const btn = document.getElementById('btn');
+const displayCurrency = document.getElementById('displayCurrency');
+
+// display option accroding to currency in rates
+const createOption = async () => {
+    try {
+        const response = await fetch('https://open.er-api.com/v6/latest/USD')
+            if(response.status !== 200 ){
+                throw new Error('something went wrong')
+            }
+                const data = await response.json()
+                const rates = data.rates; 
+                const keys = Object.keys(rates); 
+                createOptions(keys) 
+            } catch(error){
+                alert(`Sorry, there was a problem with the operation.`)
+                console.log(error.message)
+                return error.message; 
+            }
```

Comment:
More readable option IMO would be using `fetch then catch `
Good reading https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch and https://dmitripavlutin.com/javascript-fetch-async-await/
but nice with checking the response.status